glean-core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "glean-core"
-version = "19.0.0"
+version = "20.0.0-alpha.0"
```

Comment:
Uh, can we please add this [here](https://mozilla.github.io/glean/book/dev/cut-a-new-release.html) as well?

Reply:
Good point